### ISSUE TITLE
bmk won't compile

### ISSUE THREAD

**Issue by xlsior75** [bmx-ng/bmk#94]
Windows, latest BlitzmaxNG, GCC 8.10

[ 14%] Processing:bmk_cores_win32.bmx
[ 14%] Processing:stringbuffer_common.bmx
[ 15%] Processing:stringbuffer_core.bmx
[ 15%] Processing:bmk_ng_utils.bmx
[ 15%] Processing:bmk_config.bmx
[ 16%] Processing:options_parser.bmx
[ 16%] Processing:bmk_bank.bmx
[ 17%] Processing:bmk_ng.bmx
[ 17%] Processing:bmk_util.bmx
Compile Error: Overriding method does not match any overridden method. (Detail: Return type is "Void", expected "Int".)
[c:/code/BlitzMaxNG810/bmk-ng/bmk_util.bmx;1678;0]

**Comment by GWRon** [comment]
are you sure to have brl.mod updated? the definition of `TMap.Insert()` is "void" since it was made "SuperStrict" 22days ago.

**Comment by xlsior75** [comment]
Positive -- I even completely deleted the contents of brl.mod just now, and resynced it -- same error.
(compiling as x64, multi-threaded FWIW)

**Comment by xlsior75** [comment]
Nevermind -- stupid issue. It appears that my batch script to recompile the compiler was still pointing at an older version of bcc/bmk to do the actual compiling, when I switched it to use the latest one it went through without errors.

**Comment by woollybah** [comment]
Still, on a *brighter* note, there turned out to be an unrelated issue that I discovered and fixed while investigating yours :-)

**Comment by GWRon** [comment]
Good to see the issue resolved - and Brucey stumbling over other bugs meanwhile :)

**Comment by Hezkore** [comment]
I know is closed and the issue/problem was explained.
But I'm still confused as how one would solve this.
_"I switched it to use the latest one"_ how does that make sense?
If you have the latest BMK, you wouldn't have to compile a new BMK, right?

I've grabbed the latest Linux build from the blitzmax.org website.
Downloaded the new BMK source.
When I try to compile BMK I get the error from above.

What am I supposed to do?
